e2e/tests/ui-driven/src/create-flow/create-flow.spec.ts

```diff
@@ -52,166 +43,55 @@ test.describe("Navigation", () => {
-      (req) => (isRepeatedRequestMade = isGetUserRequest(req)),
-    );
-
-    Promise.all([
-      await team.click(),
-      expect(isRepeatedRequestMade).toBe(false),
-    ]);
-
-    const reloadRequest = page.waitForRequest(isGetUserRequest);
-
-    Promise.all([await page.reload(), await reloadRequest]);
-  });
-
-  test("team data persists on page refresh @regression", async ({
-    browser,
-  }) => {
-    const page = await createAuthenticatedSession({
-      browser,
-      userId: context.user!.id!,
-    });
-
-    await page.goto("/");
-    const team = page.locator("h3", { hasText: context.team.name });
-    await team.click();
-
-    const teamSlugInHeader = page.getByRole("link", {
-      name: context.team.slug,
-    });
-    await expect(teamSlugInHeader).toBeVisible();
-
-    await page.reload();
-    await expect(teamSlugInHeader).toBeVisible();
-
-    await page.goBack();
-    await expect(teamSlugInHeader).toBeHidden();
-  });
-
   test("Create a flow", async ({ browser }) => {
     const page = await getTeamPage({
       browser,
       userId: context.user!.id!,
       teamName: context.team.name,
     });
 
+    const editor = new PlaywrightEditor(page);
+
     page.on("dialog", (dialog) => dialog.accept(serviceProps.name));
-    await page.locator("button", { hasText: "Add a new service" }).click();
+    await editor.addNewService();
 
     // update context to allow flow to be torn down
     context.flow = { ...serviceProps };
 
-    const firstNode = page.locator("li.hanger > a").first();
-
-    const questionText = "Is this a test?";
-    await createQuestionWithOptions(page, firstNode, questionText, [
-      "Yes",
-      "No",
-    ]);
-    await expect(
-      page.locator("a").filter({ hasText: questionText }),
-    ).toBeVisible();
-
-    // Add a notice to the "Yes" path
-    const yesBranch = page.locator("#flow .card .options .option").nth(0);
-
-    const yesBranchNoticeText = "Yes! this is a test";
-    await createNotice(
-      page,
-      yesBranch.locator(".hanger > a"),
-      yesBranchNoticeText,
-    );
-
-    // Add a notice to the "No" path
-    const noBranch = page.locator("#flow .card .options .option").nth(1);
-    const noBranchNoticeText = "Sorry, this is a test";
-    await createNotice(
-      page,
-      noBranch.locator(".hanger > a"),
-      noBranchNoticeText,
-    );
-
-    const getNextNode = () => page.locator(".hanger > a").last();
-
-    await createChecklist(page, getNextNode(), "A checklist title", [
+    await editor.createQuestion();
+    await editor.createNoticeOnEachBranch();
+    await editor.createChecklist();
+    await editor.createTextInput();
+    await editor.createNumberInput();
+    await editor.createDateInput();
+    await editor.createAddressInput();
+    await editor.createContactInput();
+    await editor.createTaskList();
+    await editor.createFindProperty();
+    await editor.createDrawBoundary();
+    await editor.createPlanningConstraints();
```

Comment:
Per bigger picture approach questions in your PR description: 
- I'm definitely in favor of splitting out geo components into separate tests/flows as own Trello ticket suggests - makes perfect sense that draw boundary & other map-based ones are going to be a bit of a stumper here / need special handling 
  - This matches current separate tests for Pay & Send - which I wouldn't expect to necessarily be in this list either ! 
- Otherwise though I like the idea of as many component types in a single flow as we can for realistic testing - if you're finding it's very long running maybe we simply mark as `@regression` so it's picked up on the nightly run, but not per commit?